### PR TITLE
InfluxDb support

### LINKED ISSUE #87: Replace datadog usage by influxdb
As discussed in https://github.com/mozilla/release-services/issues/2248 the Mozilla Datadog account is shutting down, we need to switch to InfluxDb.

### PR DESCRIPTION
**Pull request by La0**
Fixes #87

### CODE REVIEW

File: bot/code_review_bot/stats.py [hunk 1]
```diff
@@ -3,51 +3,108 @@
...
-            "issues.{}.cleaned_paths".format(task.name), len(task.cleaned_paths)
-        )
+        self.add_metric("issues.cleaned_paths", len(task.cleaned_paths), tags)
+
+    @contextmanager
+    def timer(self, name):
+        """
+        A context manager tracking the contained code's runtime
+        """
+        start = time.time()
```
Comment (marco-c) [inline bot/code_review_bot/stats.py]: Maybe better to use https://docs.python.org/3/library/time.html#time.perf_counter ?

File: bot/tests/test_remote.py [hunk 2]
```diff
@@ -8,6 +8,31 @@
...
 from libmozdata.phabricator import UnitResultState
 
+from code_review_bot import stats
+
+
+def check_stats(summary_check):
+    """
+    Helper to check stat metrics after a workflow has run
+    """
+    # assert len(stats.metrics) == len(summary_check)
```
Comment (marco-c) [inline bot/tests/test_remote.py]: Why the commented out assertion?

Comment (La0) [reply bot/tests/test_remote.py]: Cause i'm dumb Marco.

File: bot/code_review_bot/stats.py (hunk 1 above)
Comment (La0) [reply bot/code_review_bot/stats.py]: :+1:

Comment (La0) [reply bot/code_review_bot/stats.py]: TIL

**Pull request merged**